pages/index.vue

```
@@ -60,17 +78,164 @@ const cleanSearch = computed(() => {
+    
+    ElMessage.success(`Eventos próximos a ${userCity.value} aparecerão primeiro!`);
+  } catch (error) {
+    ElMessage.error("Não foi possível obter sua localização. Verifique as permissões do navegador.");
+  }
+};
+
 const filteredEvents = computed(() => {
-  if (!cleanSearch.value) {
-    return currentEvents.value;
+  let events = currentEvents.value;
+  
```

ESLint

[error] Delete `··`

(prettier/prettier)